lib/version_utils.pm

```diff
@@ -833,6 +833,7 @@ Returns true if the SUT uses GRUB2 as bootloader
 =cut
 
 sub is_bootloader_grub2 {
+    return 0 if is_staging && check_var("VERSION", "Staging:F") && check_var('UEFI', '1') && is_tumbleweed;
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
@okurz took me les than I expected :) you can review again